### DISCUSSION TITLE
Development Status?

### DISCUSSION THREAD

**Discussion by OsparkSolutions** [wix/react-native-ui-lib#1777] [Q&A] [answered]
Is this library still undergoing active development? The typescript is very hit or miss. And the documentation is all over the place. Most behavior takes a good amount of trial and error. I like the components. But I'm trying to decide if I should move on to another library, fork this library, or help clean up some of the clutter.

If it's still being developed, I'm happy to help. Or if there is a fork that is being actively developed some direction to that would also be great. Any insight would be much appreciated.

**Comment by evg1n** [comment]
My thoughts exactly. Trying this lib for a demo project but I'm on the verge of moving back to React Native Paper.

**Comment by MuhammedKpln** [comment]
> Is this library still undergoing active development? The typescript is very hit or miss. And the documentation is all over the place. Most behavior takes a good amount of trial and error. I like the components. But I'm trying to decide if I should move on to another library, fork this library, or help clean up some of the clutter.
> 
> 
> 
> If it's still being developed, I'm happy to help. Or if there is a fork that is being actively developed some direction to that would also be great. Any insight would be much appreciated.

I have used rnuilib for a while now, and i can definitively tell that docs/ts types( for the most part ) is old or not well documented. For this reason i always finding me fiddling around the source code. Other than that i haven't found any errors preventing me from doing my job.

**Comment by ethanshar** [comment, marked as answer]
Hi, 
We are definitely in active development on this library!

I completely understand the frustration with docs and typescript but we do try to improve. 

- We recently moved to Docusaurus and started documenting our components manually to avoid missing info (by auto generating tools like react-docgen)
- Our migration to typescript has started some time ago and we already migrated most of our components to TS and we plan to keep doing so. 

If you have specific issues with our docs or missing typings for components I urge you to open an issue so we can address them accordingly. 
Moreover if you feel you can submit a PR with fixes to topics just mentioned, please do so. 

❤️
